python/UniverseWrapper.cpp

```diff
@@ -272,6 +273,42 @@ namespace FreeOrionPython {
             .def(py::map_indexing_suite<Effect::AccountingMap, true>())
         ;
 
+        //////////////////////////////////////////////
+        //   Content Named Global Constant Values   //
+        //////////////////////////////////////////////
+        py::def("namedRealDefined",
+                +[](const std::string& name) -> bool { return GetValueRef<double>(name, true); },
+                "Returns true/false (boolean) whether there is a defined double-valued "
+                "scripted constant with name (string).");
+        py::def("getNamedReal",
+                +[](const std::string& name) -> double {
+                    auto ref = GetValueRef<double>(name, true);
+                    if (!ref)
+                        return 0.0;
+                    if (ref->ConstantExpr())
+                        return ref->Eval();
+                    const ScriptingContext context;
+                    return ref->Eval(context);
+                },
+                "Returns the double-valued value of the scripted constant with name (string). "
+                "If no such named constant exists, returns 0.0.");
+        py::def("namedIntDefined",
+                +[](const std::string& name) -> bool { return GetValueRef<int>(name, true); },
+                "Returns true/false (boolean) whether there is a defined int-valued "
+                "scripted constant with name (string).");
+        py::def("getNamedInt",
+                +[](const std::string& name) -> int {
+                    auto ref = GetValueRef<int>(name, true);
+                    if (!ref)
+                        return 0;
+                    if (ref->ConstantExpr())
+                        return ref->Eval();
+                    const ScriptingContext context;
+                    return ref->Eval(context);
+                },
+                "Returns the int-valued value of the scripted constant with name (string). "
+                "If no such named constant exists, returns 0.");
+
```

Comment:
Also should python user really ever care and have to know whether it's an int or a double?
Otherwise, we could combine both for easier usage... Something like this?
```
py::object getFocsConstant(const std::string& name) {
    auto eval = [](auto&& ref) -> py::object {
        if (ref->ConstantExpr()) 
            return py::object(ref->Eval());
        return py::object(ref->Eval(ScriptingContext{}));
    };
    
    if (const auto ref = GetValueRef<double>(name, true))
        return eval(ref);
    else if (const auto ref = GetValueRef<int>(name, true))
        return eval(ref);
    else
        return py::object();
}
```